test/scrabble_test.rb

```diff
@@ -4,9 +4,55 @@
 require 'pry'
 
 class ScrabbleTest < Minitest::Test
```

Comment:
Don't forget to test your helper methods! Otherwise, this test suite is awesome.